src/app/pages/vm/vm-wizard/vm-wizard.component.ts

```diff
@@ -886,7 +886,6 @@ async customSubmit(value) {
         vm_payload["devices"].push({
           "dtype": "VNC", "attributes": {
             "vnc_port": String(this.getRndInteger(5900,65535)),
-            "vnc_resolution": "1024x768",
```

Comment:
do we want to keep it in conditionally for 12.1/freebsd? (I notice there is a backport of this too, does it make sense to port any of these changes back to 12?)

Reply:
I kept it in for 12.1 and 12 ( few lines below the above snippet). The change to middleware affects CORE and ENTERPRISE too - https://github.com/freenas/freenas/pull/5229

Reply:
I still don't see where you are keeping the resolution, what line is that on?

Reply:
It's on line 903. The conditional on 898 runs if the system isn't SCALE, adds in the non-SCALE properties of 'wait' and 'resolution'

Reply:
Oh I see now, sorry about that